src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
+    return findByXPathOrCSS(String
+            .format("(//*[contains(text(),'%s')]//following::p[contains(@class,'likersNumber')])[1]", activity));
+  }
+
+  public void clickReactionList(String activity) {
```

Comment:
The method name has to indicate which action that will be done by the user and we should avoid using event name to describe the method

```suggestion
  public void openActivityReactionsDrawer(String activity) {
```

Reply:
done